packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1551,6 +1552,64 @@ describe("normalizeAndValidateConfig()", () => {
 			});
 		});
 
+		describe("[browser]", () => {
+			it("should error if browser is an array", () => {
```

Comment:
Can you only have one browser tied to a worker? Why?

Reply:
Browsers are scarce resources that have rate limits tied to the account. Also, a single browser instance can be used for multiple tasks. Therefore it was decided that we would only allow one binding.

Reply:
That is not a use-case we would want to encourage. If the developer needs another browser in the same script, the same binding can be used for it. This was brought up before, because I had that question myself, and this was the outcome of that discussion.

Reply:
Would you be able to point me to that discussion internally?